app/Console/Services/Importer.php

```diff
@@ -423,4 +460,55 @@ protected function recalculateRanks()
     {
         /*$this->comment('--- RECALCULATING RANKS ---');*/
     }
+
+    /**
+     * Generate user's API Key and email them their new password
+     */
+    protected function setupUsers()
+    {
+        $allusers = User::all();
+        foreach($allusers as $user)
+        {
+            # Generate New User Password
+            # TODO: Increase Security?
+            $newpw = substr(md5(date('mdYhs')), 0, 10);
+
+            # Generate API Key
+            $apikey = Utils::generateApiKey();
```

Comment:
I don't mean to nitpick :) But can you change these all to `$api_key`, etc, just keep the underscores in there. I just want the variables and everything to be uniform across the code